app/src/main/java/com/framgia/forder/screen/shopinfo/productshopinfo/ProductShopInformationAdapter.java

```diff
@@ -0,0 +1,74 @@
+package com.framgia.forder.screen.shopinfo.productshopinfo;
+
+import android.content.Context;
+import android.databinding.DataBindingUtil;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import com.framgia.forder.R;
+import com.framgia.forder.data.model.Product;
+import com.framgia.forder.databinding.ItemProductShopInfoBinding;
+import com.framgia.forder.screen.BaseRecyclerViewAdapter;
+import com.framgia.forder.utils.navigator.Navigator;
```

Comment:
ERROR: Unused import: com.framgia.forder.utils.navigator.Navigator. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports